feat(about): add technologies section

tikatrsk opened the pull request:
Closes #41 

* **feat(about): add technologies section**
Added a new technologies component to showcase skill brand icons. Integrated the component into the components render MDX file. Created a technologies collection using Velite for improved data management.

* **fix(velite): update velite.config.ts**
Remove toLowerCase() this is not validate the field, its just transform the data result. Use regex() instead.

tikatrsk committed:
feat(about): Add technologies section

- Add technologies component
- Add technologies component to components render mdx
- Add technologies brand icons
- Add technologies collection velite

triskacode commented on velite.config.ts:
```diff
@@ -45,6 +45,17 @@ const personalInfo = defineSchema(() =>
   }),
 );
 
+const technologyKey = defineSchema(() => s.string().toLowerCase());
```
Remove `toLowerCase()` this is not validate the field, its just transform the data result. use regex() instead
```suggestion
const technologyKey = defineSchema(() => s.string().regex(/^[a-z0-9-.]+$/));
```

tikatrsk committed:
fix(velite): Update velite.config.ts

Remove toLowerCase() this is not validate the field, its just transform the data result.
Use regex() instead.

Co-authored-by: Triska Mahfud K <triska.triskacode@gmail.com>
